### DISCUSSION TITLE
Can you please add more Swedish channels?

### DISCUSSION THREAD

**Discussion by noobguy233** [iptv-org/community#1000] [Q&A]
Can you please add

Like
Tv 3
TV 4
TV 5
And TV 6?

Or more other Swedish channels please 🙏🏾

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
